src/Dialect/ONNX/ONNXOps.hpp

```diff
@@ -25,7 +25,7 @@
 
 namespace mlir {
 // OpSet level supported by onnx-mlir
-static constexpr int CURRENT_ONNX_OPSET = 17;
```

Comment:
@AlexandreEichenberger Are we okay with increasing this value or should we keep it at 19?

Reply:
I think that is the correct level, if one or more ops from opset 20 are supported. You can clarify this in the comments so that future developers better understand the intent.